### DISCUSSION TITLE
Verify results

### DISCUSSION THREAD

**Discussion by ArbitraryFox** [kdrag0n/freqbench#31] [Q&A] [answered]
I saw a few SoC that gets verified. How does one go about this, testing by different devs with the same SoC?

**Comment by kdrag0n** [comment, marked as answer]
Currently, verified results (low or high accuracy) are from devices that I personally own and tested. Because they're running my kernels, I can be reasonably confident that freqbench config/adaptation was done correctly and no other changes are interfering with it.

Unfortunately, I don't have the time to review everyone's kernels, so contributed results can't be verified. I can do quick preliminary reviews based on diagnostic data in freqbench results, but I'm not confident enough to consider that "verified".
